### PR TITLE
[Stylesheets] fix for https protocol for FreeCAD website

### PR DESCRIPTION
**Pull request by pgilfernandez**
Hi,

A simple fix for the FreeCAD website protocol that is `https` instead of `http`.

Cheers

### CODE REVIEW

File: src/Gui/Stylesheets/Dark-blue.qss [hunk 1]
```diff
@@ -1,8 +1,8 @@
 /*
 ABOUT
 ============================================================================================================
-version 2.20
-Qt theme (stylesheet) specially developed for FreeCAD (https://www.freecadweb.org/).
+version 2.21
+Qt theme (stylesheet) specially developed for FreeCAD (https://www.freecadweb.org//).
```
Comment (luzpaz) [inline src/Gui/Stylesheets/Dark-blue.qss]: you've added an extra `/` to the end of the URL :face_with_head_bandage: :laughing:  
You did this also for the others

File: src/Gui/Stylesheets/Light-green.qss [hunk 2]
```diff
@@ -57,7 +57,8 @@ If you would like to change the overall look/style of the theme, just find and r
 
 KNOWN BUGS and TO DO
 ============================================================================================================
-- please, follow the link to get updated information: https://forum.freecadweb.org/viewtopic.php?f=10&t=12417
+Please, follow the link to get updated information: http://forum.freecadweb.org/viewtopic.php?f=10&t=12417
```
Comment (luzpaz) [inline src/Gui/Stylesheets/Light-green.qss]: `https://` please

File: src/Gui/Stylesheets/Light-orange.qss [hunk 3]
```diff
@@ -57,7 +57,8 @@ If you would like to change the overall look/style of the theme, just find and r
 
 KNOWN BUGS and TO DO
 ============================================================================================================
-- please, follow the link to get updated information: https://forum.freecadweb.org/viewtopic.php?f=10&t=12417
+Please, follow the link to get updated information: http://forum.freecadweb.org/viewtopic.php?f=10&t=12417
```
Comment (luzpaz) [inline src/Gui/Stylesheets/Light-orange.qss]: `https://` please

File: src/Gui/Stylesheets/Dark-blue.qss [hunk 4]
```diff
@@ -57,7 +57,8 @@ If you would like to change the overall look/style of the theme, just find and r
 
 KNOWN BUGS and TO DO
 ============================================================================================================
-- please, follow the link to get updated information: https://forum.freecadweb.org/viewtopic.php?f=10&t=12417
+Please, follow the link to get updated information: http://forum.freecadweb.org/viewtopic.php?f=10&t=12417
```
Comment (luzpaz) [inline src/Gui/Stylesheets/Dark-blue.qss]: `https://` please

File: src/Gui/Stylesheets/Dark-green.qss [hunk 5]
```diff
@@ -57,7 +57,8 @@ If you would like to change the overall look/style of the theme, just find and r
 
 KNOWN BUGS and TO DO
 ============================================================================================================
-- please, follow the link to get updated information: https://forum.freecadweb.org/viewtopic.php?f=10&t=12417
+Please, follow the link to get updated information: http://forum.freecadweb.org/viewtopic.php?f=10&t=12417
```
Comment (luzpaz) [inline src/Gui/Stylesheets/Dark-green.qss]: also here

File: src/Gui/Stylesheets/Dark-orange.qss [hunk 6]
```diff
@@ -57,7 +57,8 @@ If you would like to change the overall look/style of the theme, just find and r
 
 KNOWN BUGS and TO DO
 ============================================================================================================
-- please, follow the link to get updated information: https://forum.freecadweb.org/viewtopic.php?f=10&t=12417
+Please, follow the link to get updated information: http://forum.freecadweb.org/viewtopic.php?f=10&t=12417
```
Comment (luzpaz) [inline src/Gui/Stylesheets/Dark-orange.qss]: here as well

**Comment by wwmayer** [comment]
Something is wrong with the PR because it has several conflicts.

**Comment by pgilfernandez** [comment]
Well,

I would say it's finally fixed... is it?
thanks

**Comment by pgilfernandez** [comment]
@wwmayer,
I solved the conflicts, I think everything is fine now.

**Comment by luzpaz** [comment]
@pgilfernandez you haven't fixed the `https://` for all the above **KNOWN BUGS and TO DO** forum threads.

**Comment by pgilfernandez** [comment]
Oh man, I'm dumb...
OK, I think it's done
=s

**Pull request merged**